utils/src/main/java/com/cloud/utils/StringUtils.java

```diff
@@ -78,12 +76,13 @@ public static String join(final String delimiter, final Object... components) {
         return org.apache.commons.lang.StringUtils.join(components, delimiter);
     }
 
+    /**
+     * @deprecated
+     * Please use org.apache.commons.lang.StringUtils.IsBlank() as a replacement
```

Comment:
What about referencing `org.apache.commons.lang.StringUtils.IsNotBlank` instead? So, we do not cause confusions.

Reply:
missed that one :(